server/src/serializers/RestaurantSerializer.js

```diff
@@ -6,9 +7,29 @@ class RestaurantSerializer {
     for (const attribute of allowedAttributes) {
       serializedRestaurant[attribute] = restaurant[attribute]
     }
-
     return serializedRestaurant
   }
+
+  static async getDetail(restaurant) {
+    try {
+    const allowedAttributes = ["id", "name", "location"]
+    let serializedRestaurant = {}
+
+    for (const attribute of allowedAttributes) {
+      serializedRestaurant[attribute] = restaurant[attribute]
+    }
+    const relatedBurgers = await restaurant.$relatedQuery("burgers")
+    const serializedBurgers = await Promise.all(
+      relatedBurgers.map(async (burger) => {
+        return BurgerSerializer.getSummary(burger)
+      })
+    )
+    serializedRestaurant.burgers = serializedBurgers
+    return serializedRestaurant
```

Comment:
everything in this try should be indented in

Reply:
Indentation corrected.